data-serving/scripts/converters.py

```diff
@@ -143,22 +141,22 @@ def convert_location(id: str, location_id: float, country: str, adminL1: str,
 
     try:
         if pd.notna(location_id):
-            location['id'] = float(location_id)
+            location['id'] = str(int(location_id))
```

Comment:
What does the int() conversion do, vs. just str()?

Reply:
actually not 100% sure what to do with this.

my thinking was: the existing data has integers for location ids. I'm not sure that will be the case going forward if we adopt a new geocoding system, so I want to keep the field flexible as a string, but I figured I should still throw an error if the existing data doesn't conform to the id system being used.

maybe it's best to omit this field for now and just have it in the originalCase object